SU-HSDO/suhumsci
# SHS-5988: Fix empty heading on postcards without title

# READY FOR REVIEW

## Summary
Fix empty heading on postcards without title

## Urgency
medium

## Steps to Test
1. Go to https://hs-traditional-hjupmoiykro8of7nlzyvwahjo999gbjf.tugboatqa.com/components/collections-0/collections-postcards/vertical-cards/vertical-cards-raised-cards
2. Scroll down to `Vertical Postcards with Title, Image and Link in a Collection` and `Vertical Postcards with Title and Image in a Collection` (first of the third row of cards)
3. Confirm that if there isn't a title in the card, there isn't an h2 in the markup
4. Please test this in other cards (in views too) by editing the card and removing the card title

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1209090732666835
